infrastructure/main.tf

```
@@ -52,6 +51,39 @@ module "ccpay-bulkscanning-payment-database" {
   common_tags = "${var.common_tags}"
 }
 
+# Populate Vault with DB info
+
+resource "azurerm_key_vault_secret" "POSTGRES-USER" {
+  name      = "${var.component}-POSTGRES-USER"
+  value     = "${module.ccpay-bulkscanning-payment-database.user_name}"
+  vault_uri = "${data.azurerm_key_vault.payment_key_vault.vault_uri}"
```

should use key_vault_id here instead of vault_uri which is deprecated
https://github.com/hmcts/draft-store/blob/master/infrastructure/test_config.tf#L17

```suggestion
  key_vault_id = "${data.azurerm_key_vault.payment_key_vault.id}"
```

same applies to all secrets, where was this copied from? (so we can get it updated)

---

Hi Tim,

I have done the required changes.

Can you please review and approve.

---

i've reviewed you missed some changes,
https://github.com/hmcts/ccpay-bulkscanning-app/pull/41/files/c3e172bcd43b0554d8278edbbecfb9f212c7f6b7#diff-1031eeab417cd8afe77e7e005717d38d